pkg/batchd/cache/resource_info.go

```diff
@@ -85,23 +99,25 @@ func (r *Resource) Sub(rr *Resource) *Resource {
 	if r.Less(rr) == false {
 		r.MilliCPU -= rr.MilliCPU
 		r.Memory -= rr.Memory
+		r.GPU -= rr.GPU
 		return r
 	}
 	panic("Resource is not sufficient to do operation: Sub()")
 }
 
 func (r *Resource) Less(rr *Resource) bool {
-	return r.MilliCPU < rr.MilliCPU && r.Memory < rr.Memory
+	return r.MilliCPU < rr.MilliCPU && r.Memory < rr.Memory && r.GPU < rr.GPU
 }
 
 func (r *Resource) LessEqual(rr *Resource) bool {
 	return (r.MilliCPU < rr.MilliCPU || math.Abs(rr.MilliCPU-r.MilliCPU) < 0.01) &&
-		(r.Memory < rr.Memory || math.Abs(rr.Memory-r.Memory) < 1)
+		(r.Memory < rr.Memory || math.Abs(rr.Memory-r.Memory) < 1) &&
+		(r.GPU < rr.GPU || rr.GPU == 0)
```

Comment:
@mitake I think it should be` (r.GPU < rr.GPU || r.GPU == rr.GPU)` :)

Reply:
You mean `r.GPU <= rr.GPU`?

Reply:
yes